static/locales/en-US.yaml

```diff
@@ -469,10 +469,10 @@ Settings:
     Remember History: Remember History
     Save Watched Progress: Save Watched Progress
     Save Watched Videos With Last Viewed Playlist: Save Watched Videos With Last Viewed Playlist
-    Clear Search Cache: Clear Search Cache
-    Are you sure you want to clear out your search cache?: Are you sure you want to
-      clear out your search cache?
-    Search cache has been cleared: Search cache has been cleared
+    Clear Search Cache and History: Clear Search Cache and History
```

Comment:
I could be a dumbass but when i looked at this in the FT UI i thought, "why would this remove my History". I linked this somehow to Watch History even though the button for that is just beside that. Maybe could be changed to `Clear Search Cache and Search History` or `Search Cache / History` (the latter way would be the same as with the button `Remove All Profiles / Subscription`)

Reply:
I'll try flipping it if that helps!

Reply:
LGTM thanks for unbreaking my brain